dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java

```diff
@@ -272,6 +271,12 @@ public static void createDirectoryWith755(@NonNull Path path) throws IOException
         }
     }
 
+    public static Path setOwner(@NonNull Path path, @NonNull String owner) throws IOException {
+        UserPrincipalLookupService lookupService = FileSystems.getDefault().getUserPrincipalLookupService();
+        UserPrincipal user = lookupService.lookupPrincipalByName(owner);
+        return Files.setOwner(path, user);
```

Comment:
`Files.setOwner` can only be executed by root, this is not expected.

Reply:
OK